### ISSUE TITLE
Rejected lidarr pushes are reported as a PUSH_ERROR

### ISSUE THREAD

**Issue by addisonbabcock** [autobrr/autobrr#401]
Currently (v1.3.0) when a lidarr push is rejected by lidarr those releases are reported as a PUSH_ERROR. 
![image](https://user-images.githubusercontent.com/2836233/183490961-24f46dda-8642-439a-981f-03fa2721aa17.png)

It should be reported as PUSH_REJECTED, the same as with sonarr and radarr. 
![image](https://user-images.githubusercontent.com/2836233/183491118-8c2062af-5886-4f5c-b038-740b4e59cd5b.png)

This becomes an issue when discord notifications are only enabled for PUSH_ERRORs, resulting in excessive discord pings. 

Logs of this occurring: 

```
{"level":"debug","module":"irc","network":"irc.torrentseeds.org","channel":"#announce","user":"torrentseeds","time":"2022-08-08T18:47:53Z","message":"New: VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS .:. Category: Music .:. Size: 47.81 MiB .:. URL:  https://www.torrentseeds.org/torrents/1720451 .:. Uploaded by: leaker1."}
{"level":"debug","module":"filter","time":"2022-08-08T18:47:53Z","message":"filter.Service.CheckFilter: found and matched filter: Music"}
{"level":"info","module":"release","indexer":"torrentseeds","filter":"Music","release":"VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS","time":"2022-08-08T18:47:53Z","message":"Matched 'VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS' (Music) for torrentseeds"}
{"level":"debug","repo":"release","time":"2022-08-08T18:47:53Z","message":"release.store: &{ID:27766 FilterStatus:FILTER_APPROVED Rejections:[] Indexer:torrentseeds FilterName:Music Protocol:torrent Implementation:IRC Timestamp:2022-08-08 18:47:53.895006615 +0000 UTC m=+71984.774413089 GroupID: TorrentID:1720451 TorrentURL:https://www.torrentseeds.org/torrent/download/(redacted) TorrentTmpFile: TorrentHash: TorrentName:VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS Size:50132418 Title:The 7S Collective Category:Music Season:0 Episode:0 Year:2004 Resolution: Source:WEB Codec:[] Container: HDR:[] Audio:[] AudioChannels: Group:BABAS Region: Language: Proper:false Repack:false Website: Artists:VA Type: LogScore:0 IsScene:false Origin: Tags:[] ReleaseTags: Freeleech:false FreeleechPercent:0 Bonus:[] Uploader:leaker1 PreTime: Other:[] RawCookie: AdditionalSizeCheckRequired:false FilterID:3 Filter:0xc000801400 ActionStatus:[]}"}
{"level":"error","module":"action","stack":[{"func":"(*client).Push","line":"139","source":"lidarr.go"},{"func":"(*service).lidarr","line":"62","source":"lidarr.go"},{"func":"(*service).RunAction","line":"61","source":"run.go"},{"func":"(*service).Process","line":"169","source":"service.go"},{"func":"goexit","line":"1571","source":"asm_amd64.s"}],"error":"lidarr push rejected: VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS - reasons: \"Unable to parse albums from release name\"","time":"2022-08-08T18:47:53Z","message":"lidarr: failed to push release: {VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS https://www.torrentseeds.org/torrent/download/(redacted) torrentseeds torrent torrent 2022-08-08T18:47:53Z}"}
{"level":"error","module":"action","stack":[{"func":"(*client).Push","line":"139","source":"lidarr.go"},{"func":"(*service).lidarr","line":"62","source":"lidarr.go"},{"func":"(*service).RunAction","line":"61","source":"run.go"},{"func":"(*service).Process","line":"169","source":"service.go"},{"func":"goexit","line":"1571","source":"asm_amd64.s"}],"error":"lidarr push rejected: VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS - reasons: \"Unable to parse albums from release name\"","time":"2022-08-08T18:47:53Z","message":"process action failed: push to lidarr for 'VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS'"}
{"level":"debug","module":"notification","time":"2022-08-08T18:47:53Z","message":"sending notification for PUSH_ERROR"}
{"level":"error","module":"release","indexer":"torrentseeds","filter":"Music","release":"VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS","stack":[{"func":"(*client).Push","line":"139","source":"lidarr.go"},{"func":"(*service).lidarr","line":"62","source":"lidarr.go"},{"func":"(*service).RunAction","line":"61","source":"run.go"},{"func":"(*service).Process","line":"169","source":"service.go"},{"func":"goexit","line":"1571","source":"asm_amd64.s"}],"error":"lidarr push rejected: VA-The_7S_Collective-(7SR007)-WEB-2004-BABAS - reasons: \"Unable to parse albums from release name\"","time":"2022-08-08T18:47:53Z","message":"release.Process: error running actions for filter: Music"}
```

**Comment by zze0s** [comment]
Thanks for the report. It's been fixed in #404 and will be in next release.

**Comment by addisonbabcock** [comment]
Working as expected, thanks for the fix!

![image](https://user-images.githubusercontent.com/2836233/184434516-3d7ce070-a0c9-4eef-a02c-236a9a576d7c.png)
